gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-distributionUrl=https\://services\.gradle\.org/distributions/gradle-8\.3-bin\.zip
+distributionPath=permwrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
+networkTimeout=10000
+validateDistributionUrl=true
 zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
+zipStorePath=permwrapper/dists
```

Comment:
Why change?

Reply:
I kept getting a gradle error on bump and doing `./gradlew wrapper` resolved it and did this. Not sure if it is needed. The standalone project does [it](https://github.com/wpilibsuite/StandaloneAppSamples/blob/main/Java/gradle/wrapper/gradle-wrapper.properties) to.

Reply:
lol, just curious